README.md

```diff
@@ -658,6 +658,9 @@ end
 
 ### Rodauth instance
 
+_NOTE: You should probably be using the [internal_request] feature added to
+Rodauth 2.15 instead of this API._
```

Comment:
Could the existing API be made to use `internal_request_eval` perhaps?

Reply:
Hmm, interesting 🤔 You're right, I know see we can get the instance for example by using `Object#itself`:

```rb
rodauth.request_instance_eval(&:itself)
```

That would require rodauth-rails to load the internal_request feature automatically though. Or I could make a breaking change where it fails if the internal_request feature isn't loaded, since rodauth-rails is still in the 0.x phase.

Reply:
Yeah, both good options. Either way, it's an alternative to _removing_ this feature of the library.

Reply:
I've implemented `Rodauth::Rails.rodauth` using `internal_request_eval`, but kept backwards compatibility by auto-enabling the internal_request feature. I've deprecated `:query` and `:form` in favour of `:params`, but I've kept `:account` because it automatically sets `@account` which is required by some Rodauth methods.

It's good that this way we get to keep easy access to the Rodauth instance.